digicampipe/calib/camera/dl2.py

```diff
@@ -19,25 +19,19 @@ def calibrate_to_dl2(event_stream, reclean=False, shower_distance=80*u.mm):
 
             mask = dl1_camera.cleaning_mask
             image[~mask] = 0.
-
-            try:
-
-                moments_first = hillas.hillas_parameters(geom, image)
-
-                if reclean:
-
-                    mask_near_center = find_mask_near_center(geom=geom,
-                                                             cen_x=moments_first.cen_x,
-                                                             cen_y=moments_first.cen_y,
-                                                             distance=shower_distance)
-                    dl1_camera.cleaning_mask = dl1_camera.cleaning_mask & mask_near_center
-                    image[~dl1_camera.cleaning_mask] = 0
-                    moments = hillas.hillas_parameters(geom, image)
-                else:
-                    moments = moments_first
-            except HillasParameterizationError:
-                print('could not recompute Hillas')
-                moments = None
+            moments_first = hillas.hillas_parameters(geom, image)
```

Comment:
I don't get why the `try` --> `except` was removed. what if we get a HillasParameterizationError?

Reply:
2 Reasons:
 1. HillasParameterizationError was a SyntaxError. "HillasParameterizationError" was simply unknown. So once the except was reached a SyntaxError was raised. 
 2. When `moments=None` later code is trying to do access `moments.size.shape` or so .. I do not remember. That did not work since moments was None, and None has no attribute `.size` or however it was called.

So I caught the exception deeper down in `hillas.hillas_parameters` ... but obviously I did not push that file ... damn it.

Reply:
Ok but we should still have an except here. Since sometimes the `hilllas_parameters()` function can fail in that case we don't want the pipeline to stop. So maybe setting it to None is wrong but we rather set it a 0/nan `MomentParameters` container ?

Reply:
Or maybe just jump to the next iteration with `continue` ?

Reply:
Oh! I know why I did not push the file .. It is in ctapipe.

Reply:
Ok now I did not fix it in ctapipe, but in digicampipe.

Reply:
I got confused about what is what. Sometimes I see something funny and want to fix it, and then I see it is part of "CTS" .. this is confusing for me.